src/loader.c

```diff
@@ -301,7 +301,7 @@ static void ldr_set_encoding(struct fmt_main *format)
 
 	/* john.conf alternative for --internal-encoding */
 	if (options.flags &
-	    (FLG_RULES | FLG_SINGLE_CHK | FLG_BATCH_CHK | FLG_MASK_CHK))
+	    (FLG_RULES | FLG_SINGLE_CHK | FLG_BATCH_CHK | FLG_MASK_CHK | FLG_FUZZ_CHK))
```

Comment:
I add this because there is a bug. But I can not reproduce now. I will remove this change. If the bug  comes back, I will fix it then.